backend/core/src/auth/dto/user-profile.dto.ts

```diff
@@ -23,6 +23,7 @@ export class UserProfileUpdateDto extends PickType(User, [
   "createdAt",
   "updatedAt",
   "language",
+  "phoneNumber",
```

Comment:
Trying to understand the reason for adding phoneNumber here - this doesn't impact what UserService.findByEmail returns, does it?

Reply:
This is so that I could update the user phone number through the swagger docs. We'll eventually want this anyways, when we add a PhoneField to the edit account page.